hera_pspec/plot.py

```diff
@@ -494,3 +496,176 @@ def delay_waterfall(uvp, blpairs, spw, pol, component='real', average_blpairs=Fa
     if new_plot:
         return fig
     
+def delay_wedge(uvps, blpairs, fold=False, cosmo=True, center_line=True, horizon_lines=True, cosmo_cbar=True, suptitle=''):
+    """
+    Plot a 2D delay spectrum (or spectra) for a group of baselines.
+    
+    Parameters
+    ----------
+    uvps : UVPspec
+        List of UVPSpec objects, containing delay spectra for a set of baseline-pairs, 
+        times, polarizations, and spectral windows.
+        
+    blpairs : list of tuples
+        List of baseline-pair tuples.
+        
+    fold : bool, optional
+        Whether to fold the power spectrum in :math:`|k_\parallel|`. 
+        Default: False.
+        
+    cosmo : bool, optional
```

Comment:
this kwarg `cosmo` is a little confusing b/c we also call `cosmo` an object that holds cosmologies. Let's change this to `delay`, which is `True` if plotting tau and `False` if plotting k